src/smt/proof_post_processor.cpp

```diff
@@ -842,15 +843,16 @@ Node ProofPostprocessCallback::expandMacros(PfRule id,
     Assert(!args.empty());
     Node eq = args[0];
     Assert(eq.getKind() == EQUAL);
-    ProofNodeManager* pnm = d_env.getProofNodeManager();
-    // try to replay theory rewrite
-    // first, check that maybe its just an evaluation step
-    ProofChecker* pc = pnm->getChecker();
-    Node ceval = pc->checkDebug(
-        PfRule::EVALUATE, {}, {eq[0]}, Node::null(), "smt-proof-pp-debug");
-    if (!ceval.isNull() && ceval == eq)
+    TheoryId tid = THEORY_BUILTIN;
+    builtin::BuiltinProofRuleChecker::getTheoryId(args[1], tid);
+    MethodId mid = MethodId::RW_REWRITE;
+    getMethodId(args[2], mid);
+    uint32_t recLimit = options().proof.proofRewriteRconsRecLimit;
```

Comment:
The type in the option declaration is `int64_t`. Shouldn't these match?

Reply:
Yes, this must have gotten out of sync.